packages/block-editor/src/components/inserter/style.scss

```diff
@@ -161,6 +161,7 @@ $block-inserter-search-height: 38px;
 		padding-bottom: 20px;
 		margin-bottom: 10px;
 		border-bottom: $border-width solid $light-gray-500;
+		@include edit-post__fade-in-animation();
```

Comment:
Why is this animation called `edit-post` ? Is this in the global mixin, should we give it a generic name?

Reply:
Yeah, it should — we use it in many different places. I'll open a separate PR for that.